src/platforms/javascript/common/session-replay/privacy.mdx

```diff
@@ -69,6 +69,31 @@ new Sentry.Replay({
 });
 ```
 
+### Example: Capturing 500 Status Codes Only
+
+Here is an example that will only capture fetch requests that return a 500 status code. Non-fetch requests will continue to be captured as normal.
+
+```javascript
+new Sentry.Replay({
+  beforeAddRecordingEvent: (event) => {
+    // Capture non-fetch requests as normal
+    if (
+      event.data.tag !== "performanceSpan" ||
+      event.data.payload.op !== "resource.fetch"
+    ) {
+      return event;
+    }
+
+    // Only capture requests that result in a 500 status code
+    if (event.data.payload.data.statusCode !== 500) {
+      return null;
+    }
+
+    return event;
```

Comment:
IMHO The negation in the first stanza, and duplicate `return event` statements, make this harder to understand if someone's not as familiar with the types.

what about this:

```suggestion
    // Do not capture fetch/xhr requests, unless the response code is 500
    if (
      event.data.tag === "performanceSpan" &&
      (event.data.payload.op === "resource.fetch" || event.data.payload.op === 'resource.xhr') &&
      event.data.payload.data.statusCode !== 500
    ) {
      return null;
    }
    
    return event;
```

Reply:
Yeah this makes more sense.